core/src/test/java/org/bitcoinj/core/BlockTest.java

```diff
@@ -199,45 +202,46 @@ public void testCoinbaseHeightTestnet() throws Exception {
         block.verify(5000, EnumSet.of(Block.VerifyFlag.HEIGHT_IN_COINBASE));
     }
 
-    /*@Test
+    @Test // This needs to be updated for Dash
```

Comment:
This test was previously commented out.  Now it can be used with data from the Testnet blockchain.